recipe/meta.yaml

```diff
@@ -1,20 +1,18 @@
-{% set name = "fastjsonschema" %}
-{% set version = "2.15.1" %}
+{% set version = "2.16.1" %}
 
 package:
-  name: python-{{ name }}
+  name: python-fastjsonschema
   version: {{ version }}
 
 source:
   - folder: dist
-    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-    sha256: 671f36d225b3493629b5e789428660109528f373cf4b8a22bac6fa2f8191c2d2
+    url: https://pypi.io/packages/source/f/fastjsonschema/fastjsonschema-{{ version }}.tar.gz
+    sha256: d6fa3ffbe719768d70e298b9fb847484e2bdfdb7241ed052b8d57a9294a8c334
   - folder: src
-    url: https://github.com/horejsek/python-{{ name }}/archive/v{{ version }}.tar.gz
-    sha256: 1a59d7ca91c90b79160ad43859caddb699cad07ddbb91c364a15b1b18f3a1fc4
+    url: https://github.com/horejsek/python-fastjsonschema/archive/v{{ version }}.tar.gz
+    sha256: 1acdf6bfaa4a16cbb5826f8abba40c6ac896a860c51b87edb2acbfb9462a2858
 
 build:
-  noarch: python
   number: 0
   script: cd dist && {{ PYTHON }} -m pip install . -vv
 
```

Comment:
We might want to add the `python` pinnings in the `skip section` instead of adding it in the `host` and `run` sections.

Reply:
And could you please add `setuptools` and `wheel` in `host` explicitly? If you check logs you can find that both are available in `host` implicitly